### PR TITLE
canvas-embed-node npm package

### PR DESCRIPTION
**Pull request by JeremyBernier**
- npm package `canvas-embed-node`
- `nodejs-example` folder in repo demonstrating usage of this package

The npm package has been published

### CODE REVIEW

File: canvas-embed-node/build/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"use strict";
```
Comment (wpride) [inline canvas-embed-node/build/index.js:1]: should gitignore everything in `/build`

File: canvas-embed-node/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```
Comment (wpride) [inline canvas-embed-node/tsconfig.json:3]: Why all the commented out lines in here?

File: nodejs-example/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nodejs-example",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "To generate Canvas authentication tokens from NodeJS you can use the following snippet:",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (wpride) [inline nodejs-example/package.json:8]: kill?

Comment (wpride) [reply nodejs-example/package.json:8]: and maybe add a `run` script?

File: canvas-embed-node/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import _sodium from "libsodium-wrappers";
+import btoa from "btoa";
+
```
Comment (wpride) [inline canvas-embed-node/src/index.ts:3]: Doc string here would probably be good

File: canvas-embed-node/package.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "canvas-embed-node",
+  "version": "1.0.0",
+  "description": "Canvas embed package for Node.js",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc"
```
Comment (wpride) [inline canvas-embed-node/package.json:7]: How will publishing work?

**Review by wpride — CHANGES_REQUESTED** [review]

File: canvas-embed-node/tsconfig.json (hunk 2 above)
Comment (JeremyBernier) [reply canvas-embed-node/tsconfig.json:3]: I used the default `tsconfig.json` file generated by `tsc --init` as a starting point. Do you want me to delete them?

File: canvas-embed-node/package.json (hunk 5 above)
Comment (JeremyBernier) [reply canvas-embed-node/package.json:7]: Can add CI/CD to this PR

File: canvas-embed-node/src/index.ts (hunk 4 above)
Comment (JeremyBernier) [reply canvas-embed-node/src/index.ts:3]: just added

File: canvas-embed-node/package.json (hunk 5 above)
Comment (JeremyBernier) [reply canvas-embed-node/package.json:7]: added

File: nodejs-example/package.json (hunk 3 above)
Comment (JeremyBernier) [reply nodejs-example/package.json]: assume you mean `start` script to run `node index.js`, done

File: canvas-embed-node/tsconfig.json (hunk 2 above)
Comment (JeremyBernier) [reply canvas-embed-node/tsconfig.json]: Ok removed (per Slack conversation)

File: canvas-embed-node/build/index.js (hunk 1 above)
Comment (JeremyBernier) [reply canvas-embed-node/build/index.js]: ok done

**Review by wpride — APPROVED** [review]

**Pull request merged**